### PR TITLE
Added Docyard application as Not Vulnerable

### PR DESCRIPTION
**Pull request by fransbloemen**
Thank you for your contribution! Some pointers to get it merged quickly:

For contributions in `software/`:

  - Status: please select a value from the status table at the top
  - Version: Status Vulnerable / Workaround? -> List vulnerable versions.
             Status Fix?                     -> List fixed versions.
  - Links: make sure you link to a public statement by the developer.
    Alternatively, include and link a file in the `software/vendor-statements/`
    directory
  - Please mind the sorting
  - Please put vendor/product name in PR title (instead of "Update README.md")

### CODE REVIEW

File: software/README.md [hunk 1]
```diff
@@ -738,6 +737,7 @@ The `Version` relates to the `Status` column. If `Status` is Vulnerable, Version
 | Dell | Wyse Windows Embedded | Unknown | Vulnerable | Fix Release Timeline TBD | [source](https://www.dell.com/support/kbdoc/nl-nl/000194414/dell-response-to-apache-log4j-remote-code-execution-vulnerability) |
 | Devolutions | All products | | Not vuln | | [source](https://blog.devolutions.net/2021/12/critical-vulnerability-in-log4j/) | 
 | Docker | Docker infrastructure | Unknown | Not vuln | Docker infrastructure not vulnerable, Docker images could be vulnerable. For more info see source. |[source](https://www.docker.com/blog/apache-log4j-2-cve-2021-44228/) |
+| Docuwork | DocYard | All Versions | Not Vuln | | [source](https://docuwork.nl) |
```
Comment (maertsen) [inline software/README.md:740]: please provide a more specific source url; the homepage does not contain any log4j information at a first glance.

File: software/README.md [hunk 2]
```diff
@@ -88,7 +88,6 @@ The `Version` relates to the `Status` column. If `Status` is Vulnerable, Version
 | Apache        | James | 3.6.0 | Vulnerable |  |[source](https://twitter.com/dlitchfield/status/1469809966785564675) |
 | Apache        | Jena | < 4.3.1 | Fix | Fixed in 4.3.1 |[source](https://blogs.apache.org/security/entry/cve-2021-44228)|
 | Apache        | JMeter | Any | Vulnerable| Manual Bypass |[source](https://github.com/apache/jmeter/pull/680#issuecomment-993204248)|
-| Apache        | JSPWiki | 2.11.1 | Fix | |[source](https://jspwiki-wiki.apache.org/Wiki.jsp?page=Log4J-CVE-2021-44228) |
```
Comment (maertsen) [inline software/README.md:91]: Can you restore this line?

**Review by maertsen — CHANGES_REQUESTED** [review]
hi @fransbloemen, thanks for your contribution. Can you take a look at my comments?

File: software/README.md (hunk 1 above)
Comment (fransbloemen) [reply software/README.md:741]: We'll do that soon!

File: software/README.md (hunk 2 above)
Comment (fransbloemen) [reply software/README.md]: Done

File: software/README.md (hunk 1 above)
Comment (maertsen) [reply software/README.md:741]: I would prefer a more stable url than the homepage, say a direct link to a news item. I'll merge once your provide one.

**Review by maertsen — CHANGES_REQUESTED** [review]

**Comment by maertsen** [comment]
hi @fransbloemen, I'm closing this PR for lack of response. Feel free to reopen one when you have an updated url for us.

**Pull request closed without merging**